aip/general/0126.md

```diff
@@ -55,12 +55,18 @@ message Book {
   - An exception to this rule is if there is a clearly useful zero value. In
     particular, if an enum needs to present an `UNKNOWN`, it is usually clearer
     and more useful for it to be a zero value rather than having both.
+- The other values **should not** be prefixed by the name of the enum itself.
+  This generally requires users to write `MyState.MYSTATE_ACTIVE` in their
+  code, which is unnecessarily verbose.
+  - Note that some languages (including C++) hoist enum values into the parent
+    namespace, which can result in conflicts for enums with the same values. To
+    avoid this, multiple enums in the same namespace **must not** share any
+    values. To avoid sharing values, APIs **may** prefix enum values with the
+    name of the enum. In this case, they should do so consistently within the
```

Comment:
Should we use a requirement word in this branch? `In this case, they **must** do so consistently within the enum.`

Reply:
Yes - in fact, I meant to do that in the last revision (sorry, Jon)! Changed to `**must**`.